src/components/Overlays/OverlayHeaderUsername.tsx

```diff
@@ -0,0 +1,33 @@
+import { ServiceCredentialBindingObj } from '@/api/cf/cloudfoundry-types';
+import { Username } from '@/components/UserAccount/Username';
+import { Tag } from '@/components/uswds/Tag';
+
+export function OverlayHeaderUsername({
+  header,
+  serviceAccount,
+  username,
+}: {
+  header: string;
+  serviceAccount?: ServiceCredentialBindingObj | undefined | null;
+  username: string;
+}) {
+  return (
+    <>
+      <h4
+        className="margin-top-0 margin-bottom-7 text-uppercase text-light underline-base-light text-underline"
+        style={{ textUnderlineOffset: '0.7em' }}
```

Comment:
[non-blocking comment, TIL] I actually didn’t know about `text-underline-offset`! (I would’ve just used border-bottom, and offset it with a little padding.)

Reply:
Me neither until recently! It seemed to have [good enough browser support](https://developer.mozilla.org/en-US/docs/Web/CSS/text-underline-offset#browser_compatibility) (if you're not using percentage values) so I thought I'd use it.